lib/serializer.ts

```diff
@@ -618,6 +622,27 @@ function deserializeCompositeType(serializer: Serializer, mapper: CompositeMappe
         instance[responsePropName] = serializer.deserialize(additionalPropertiesMapper, responseBody[responsePropName], objectName + '["' + responsePropName + '"]');
       }
     }
+  } else {
+    const containsSerializedName = (modelProperties: {[propertyName: string]: Mapper}, serializedKey: string): boolean => {
+      for (const key of Object.keys(modelProperties)) {
```

Comment:
nit: `Object.keys(modelProperties).find(…)`

Reply:
I don't agree with this suggestion. I think the foreach pattern is just fine here.